api/lib/schema.js

```diff
@@ -110,6 +110,7 @@ const typeDefs = gql`
     beneficiary: NetworkAccount
     summary: String
     parameter: String
+    changes: Changes
```

Comment:
maybe:
```suggestion
    changes: [Changes]
```

Reply:
```suggestion
    changes: [Change]
```
But not, actually just String it is the easiest. We just want to display a stringified JSON here. It is only a display thing after, we don't need another type to treat this data